lib/facter/rhsm.rb

```diff
@@ -0,0 +1,47 @@
+Facter.add(:rhsm, type: :aggregate) do
+  confine :os do |os|
+    os['name'] == 'RedHat'
```

Comment:
What about CentOS? subscription-manager is also available there.

Reply:
Hmm, what would the point be? Can CentOS use subscription-manager to work with custom satellite repos?

I was trying to exclude it to prevent my CentOS machines from failing because they don't have subscription-manger installed by default.

Reply:
No big deal either way - I've updated the calculation of the facts so they won't break on machines which don't have subscription-manager installed.